### PR TITLE
[FEAT] Demo time/frequency monitoring: Improve button style

### PR DESCRIPTION
**Pull request by csouchet**
**Live environment of examples**

https://cdn.statically.io/gh/process-analytics/bpmn-visualization-examples/monitoring_demo/improve_button_css/examples/index.html

### CODE REVIEW

**Comment by csouchet** [comment]
### Buttons & title on same line 
image

### Buttons & title on different line

**Review by tbouffard — APPROVED** [review]
LGTM, few minor things that could be improved now or in a dedicated refactoring PR

File: demo/monitoring-all-process-instances/js/index.js [hunk 1]
```diff
@@ -45,27 +41,28 @@ function switchDiagram(switchValue) {
...
-    console.info('Switched to %s', currentDiagram)
+    console.info('Switched to %s', switchValue)
 }
 
 document.addEventListener('DOMContentLoaded', function () {
     updateTimeLegends();
 })
 
-function switchData(bpmnVisualization, data) {
+function switchData() {
```
[the remark this answers is not in the document]
Comment (csouchet) [reply demo/monitoring-all-process-instances/js/index.js:59]: I'll rename the function in the PR for the refactoring 😉 #228

File: demo/monitoring-all-process-instances/js/index.js [hunk 2]
```diff
@@ -36,6 +31,7 @@ let frequencyData;
 document.getElementById('choose-diagram-panel').onclick = () => {
     const diagramType = document.querySelector("input[type='radio'][name='diagram-type']:checked").value;
     switchDiagram(diagramType);
+    switchData();
 }
 
 function switchDiagram(switchValue) {
```
[the remark this answers is not in the document]
Comment (csouchet) [reply demo/monitoring-all-process-instances/js/index.js:37]: I'll rename the function in the PR for the refactoring 😉 #228

**Pull request merged**